Runtime/editor/editor_mouse_tracking.cs

```diff
@@ -0,0 +1,189 @@
+using System.Collections.Generic;
+using Unity.UIWidgets.foundation;
+using Unity.UIWidgets.scheduler;
+using UnityEditor;
+
+namespace Unity.UIWidgets.gestures {
+    public partial class MouseTracker {
+        bool _enableDragFromEditorRelease = false;
+
+        void _handleDragFromEditorEvent(PointerEvent evt, int deviceId) {
+            if (evt is PointerDragFromEditorReleaseEvent) {
+                this._enableDragFromEditorRelease = false;
+                this._scheduleDragFromEditorReleaseCheck();
+                this._lastMouseEvent.Remove(deviceId);
+            }
+            else if (evt is PointerDragFromEditorEnterEvent ||
+                     evt is PointerDragFromEditorHoverEvent ||
+                     evt is PointerDragFromEditorExitEvent) {
+                if (!this._lastMouseEvent.ContainsKey(deviceId) ||
+                    this._lastMouseEvent[deviceId].position != evt.position) {
+                    // Only schedule a frame if we have our first event, or if the
```

Comment:
maybe we can remove this comment, since the information within is not that hard to understand from the code itself

Reply:
the same for some comments in other places too. Please try to remove all unnecessary comments in this PR